src/Components/Shared/Loader/index.jsx

```diff
@@ -0,0 +1,6 @@
+import styles from './loader.module.css';
+import React from 'react';
+
+export function Loader() {
```

Comment:
Please add default, to directly export it.
```suggestion
export default function Loader()
```